src/styles/form.scss

```diff
@@ -0,0 +1,51 @@
+.form {
+  width: 40%;
+
+  @media (max-width: $displayLaptop) {
+    margin-top: 32px;
+    width: auto;
+  }
+
+  &__input {
+    resize: vertical;
```

Comment:
remove this

Reply:
No! I want to user cant resize textarea. And I dont want to make special class for only one property

Reply:
if you want to assign some value to one element, it's completely weird if you assign it to all the elements just because you don't want to create extra class

use only necessary files, there is no place for 'just in case'

Reply:
ok